printer/testdata/raw_string.in

```diff
@@ -0,0 +1,15 @@
+block `label` {
+	attr = `'\"attr`
+}
```

Comment:
Hm, I wonder if we should even support this for labels, or if labels should require the double quoted syntax to start with.